src/ontology/mondo-edit.obo

```diff
@@ -82,6 +82,7 @@ property_value: IAO:0000700 MONDO:0000001
 property_value: IAO:0000700 MONDO:0021125
 property_value: IAO:0000700 MONDO:0042489
 property_value: MONDO "0015163format-version: 1.2" xsd:string
+owl-axioms: Prefix(owl:=<http://www.w3.org/2002/07/owl#>)\nPrefix(rdf:=<http://www.w3.org/1999/02/22-rdf-syntax-ns#>)\nPrefix(xml:=<http://www.w3.org/XML/1998/namespace>)\nPrefix(xsd:=<http://www.w3.org/2001/XMLSchema#>)\nPrefix(rdfs:=<http://www.w3.org/2000/01/rdf-schema#>)\n\n\nOntology(\nDeclaration(Class(<http://purl.obolibrary.org/obo/MONDO_0005734>))\nDeclaration(Class(<http://purl.obolibrary.org/obo/NCBITaxon_9793>))\nDeclaration(Class(<http://purl.obolibrary.org/obo/NCBITaxon_9796>))\nDeclaration(ObjectProperty(<http://purl.obolibrary.org/obo/RO_0002162>))\nDeclaration(AnnotationProperty(<http://www.geneontology.org/formats/oboInOwl#source>))\n\n\n\n############################\n#   Classes\n############################\n\n# Class: <http://purl.obolibrary.org/obo/MONDO_0005734> (dourine)\n\nSubClassOf(Annotation(<http://www.geneontology.org/formats/oboInOwl#source> \"https://orcid.org/0000-0002-4142-7153\"^^xsd:string) Annotation(<http://www.geneontology.org/formats/oboInOwl#source> \"https://orcid.org/0000-0002-5002-8648\"^^xsd:string) <http://purl.obolibrary.org/obo/MONDO_0005734> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/RO_0002162> ObjectUnionOf(<http://purl.obolibrary.org/obo/NCBITaxon_9793> <http://purl.obolibrary.org/obo/NCBITaxon_9796>)))\n\n\n)
```

Comment:
Something wrong happened here. Best guess: Problem with nesting in the OWL class expression of MONDO_0005734